src/transformers/modeling_tf_utils.py

```diff
@@ -1065,3 +1065,33 @@ def cast_bool_to_primitive(bool_variable: Union[tf.Tensor, bool], default_tensor
 
     # else variable is bool
     return bool_variable
+
+
+class NoLayerEmbedTokens:
```

Comment:
Tough to name well.
WDYT @patrickvonplaten ?
maybe 

```python
TFEmbeddingsPointer
	"""Wraps TFSharedEmbeddingTokens to avoid a problem with weight restoring."""
```
or maybe `TFWrappedEmbeddings`, `TFCopiedEmbeddings`?

Reply:
`TFEmbeddingsHack`

Reply:
Hmm, so the whole idea of this class is to wrap the embeddings into something which *is not* recognized as a weight by TF so that one can restore the model correctly and convert it from PT. 

=> I'm having a hard time finding a good name. I think I like `TFWrappedEmeddings` the best.

Reply:
did `TFWrappedEmbeddings`